packages/core/src/components/AccordionGroup/Accordion.tsx

```diff
@@ -0,0 +1,31 @@
+import CardBase from '@components/Card/CardBase';
+import { AccordionProps } from './types';
+import { createWrapperStyles } from './styles';
+
+const Accordion = ({
+  title,
+  isActive,
+  ariaControls,
+  tabId,
+  variant = 'empty',
+  renderTitle,
+  renderContent,
+  onClick,
+}: AccordionProps) => {
+  const styles = createWrapperStyles();
```

Comment:
Let's move it outside of the component so that the object with styles is created once.

Reply:
Done